wiseconnect/components/device/silabs/si91x/wireless/inc/sl_wifi_device.h

```diff
@@ -302,7 +302,7 @@
 /// @note Bit 30 is reserved
 
 /// Extention valid to use Extended custom feature bitmap
-#define SL_SI91X_CUSTOM_FEAT_EXTENTION_VALID BIT(31)
+#define SL_SI91X_CUSTOM_FEAT_EXTENSION_VALID BIT(31)

```

Comment:
Is this purely a downstream change in this tree, or was the same fix also submitted to the upstream wiseconnect tree? If it was not submitted upstream, we'll need to keep making sure to always fix this as we move to newer wiseconnect versions. Btw, the code comment above this line still says "Extention" :)

Reply:
It's a pure downstream change, we could raise a ticket to WiseConnect team to fix this.

This typo is only present in the validity bits (which is currently the most significant bit).
So another possible solution would be to create new defines in the Zephyr repo with "BIT(31)". However if the SDK changes this can cause a "silent error" whereas the typo fix will result in a build error.

What would be your preference?

Reply:
We want to automate the importing of Silabs SDKs into `hal_silabs`. @asmellby has already done the initial work for the SiSDK in his latest PR. I think the cleanest and most sustainable option would be to raise a ticket to get this fixed upstream, since otherwise we would need to have extra logic in our automation to do the fix there. The option of having a custom define in Zephyr is the least preferred one, since we want to stay in sync with the upstream SDK and its API.

Reply:
Okay, I'll leave it as is now and create a ticket about this.